Dockerfile

```diff
@@ -14,10 +14,11 @@ RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
 ENV PATH /opt/conda/bin:$PATH
 ENV PYTHONPATH /gale:$PYTHONPATH
 
-#Create gale conda environment (like cd gale)
+# Create gale conda environment (like cd gale)
 WORKDIR /gale
 ADD environment.yml .
 RUN conda env create -f environment.yml && conda clean -a -y
+RUN conda init bash
```

Comment:
what is the advantage if this w.r.t of not having it?

Reply:
For me conda did not work without this line. I had to run it every time I entered the container.